tests/de/scene_HassTurnOn.yaml

```diff
@@ -0,0 +1,13 @@
+language: de
+tests:
+  - sentences:
```

Comment:
Please add test sets for each possible variant

Reply:
i count 15 possible variants, but only 7 test sentences 😁

Reply:
You mean for every combination which each of the expansion rules? Shouldn't it be fine to have tests for each line of the defined sentences?

Reply:
We should have tests for every possible variant, so we can ensure that later changes do not brake already working sentences (_avoid regressions_)

Reply:
Alright, makes sense. Found some more 😄